src/routes/index.js

```diff
@@ -1,4 +1,7 @@
 const express = require("express");
 const router = express.Router();
 
-module.exports = router;
+const getRouter = require("./getRouter");
```

Comment:
Router 이름이 getRouter가 아니라 자원 기준으로 ProductRouter가 되도록 수정해주세요.

```suggestion
const getRouter = require("./getRouter");
```